profiles/databricks.yaml

```diff
@@ -1,27 +1,33 @@
-# [Recommended] Uncomment fields below to set an MLflow experiment to track the pipeline execution.
-# If unset, a default experiment based on runtime context will be created.
-# experiment:
-  # name: "/Shared/sklearn_regression_experiment"
-
-# Set the registry server URI. This property is especially useful if you have a registry
-# server that’s different from the tracking server.
-# Profile could be created using https://github.com/databricks/databricks-cli#installation
+#
+# FIXME::REQUIRED: set an MLflow experiment name to track pipeline executions and artifacts.
```

Comment:
Shall we include optional configs for tracking URI / artifact location?

Reply:
On Databricks? Do you mean sending tracking info to a different shard? I'd say it's an advanced option. We can mention it in the user guide but I don't think it's necessary to list it here.